### PR TITLE
product service change

### PR DESCRIPTION
**Pull request by sindhura-pamujula**

### CODE REVIEW

File: auth-ms/src/app.ts [hunk 1]
```diff
@@ -96,7 +97,7 @@ app.get("/signup", userController.getSignup);
 app.post("/signup", userController.postSignup);
 app.get("/contact", contactController.getContact);
 app.post("/contact", contactController.postContact);
-app.get("/account", passportConfig.isAuthenticated, userController.getAccount);
+app.get("/account",passportConfig.isAuthenticated, userController.getAccount);
 app.post("/account/profile", passportConfig.isAuthenticated, userController.postUpdateProfile);
 app.post("/account/password", passportConfig.isAuthenticated, userController.postUpdatePassword);
 app.post("/account/delete", passportConfig.isAuthenticated, userController.postDeleteAccount);
```
Comment (SandeepPamujula) [inline auth-ms/src/app.ts:103]: Move these lines to different routes files.

File: auth-ms/src/config/passport.ts [hunk 2]
```diff
@@ -40,7 +44,40 @@ passport.use(new LocalStrategy({ usernameField: "email" }, (email, password, don
     });
 }));
 
+const options = {
+    jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+    secretOrKey: "secret"
+  };
+passport.use(new JwtStrategy(options, function(jwtPayload: any, done: any) {
 
+    console.log(jwtPayload);
```
Comment (SandeepPamujula) [inline auth-ms/src/config/passport.ts:53]: remove console logs. use any logger module

File: products/src/app.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        const PORT = process.env.PORT || 4000;
+        this.app.listen(PORT,()=>console.log(`products server running on ${PORT}`));
+
+    }
+
+    private config(): void {
+        
+        // Connect to MongoDB
+        mongoose.Promise = bluebird;
+        const mongoUrl= "mongodb+srv://sindhu:sindhu1234@customers-9xbiq.mongodb.net/products?retryWrites=true&w=majority";
```
Comment (SandeepPamujula) [inline products/src/app.ts:24]: sensitive info should come from .env file

File: products/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    // "lib": [],                             /* Specify library files to be included in the compilation. */
+    // "allowJs": true,                       /* Allow javascript files to be compiled. */
+    // "checkJs": true,                       /* Report errors in .js files. */
+    // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+    // "declaration": true,                   /* Generates corresponding '.d.ts' file. */
+    // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
+    // "sourceMap": true,                     /* Generates corresponding '.map' file. */
```
Comment (SandeepPamujula) [inline products/tsconfig.json:15]: enable sourceMap so that we can debug

File: auth-ms/src/config/passport.ts (hunk 2 above)
Comment (sindhura-pamujula) [reply auth-ms/src/config/passport.ts:53]: ok

**Comment by sindhura-pamujula** [comment]
merged from productbranch

**Pull request merged**